app/models/draft-registration.ts

```diff
@@ -63,6 +63,15 @@ export default class DraftRegistrationModel extends OsfModel {
 
     @hasMany('contributor')
     contributors!: DS.PromiseManyArray<ContributorModel> & ContributorModel[];
+
+    get currentUserIsAdmin() {
+        return Array.isArray(this.currentUserPermissions) && this.currentUserPermissions.includes(Permission.Admin);
+    }
+
+    get currentUserIsReadOnly() {
```

Comment:
using a native getter here (without computed) means the value is computed every time the property is accessed (so no caching). The computation here is cheap so it's probably not a big deal.